src/modules/map/components/toolbox/TimeSlider.vue

```diff
@@ -409,6 +437,15 @@ function setYearToInputIfValid() {
                     </small>
                 </div>
             </div>
+
+            <div class="time-slider-dropdown" data-cy="time-slider-dropdown">
+                <select v-model="currentYear" class="form-select ml-2">
+                    <option v-for="year in allYears" :key="year" :value="year">
+                        {{ year }}
+                    </option>
+                </select>
+            </div>
```

Comment:
Here it would be better/nicer to have an input-group with the play button something similar to 
![image](https://github.com/geoadmin/web-mapviewer/assets/67745584/e0064e36-3d22-466c-9f3f-536eb273b4ab)

About the search feature for the year it is a good idea and we can use the same logic as for the external layer provider dropdown, so the user could enter the year and that would filter, see how the external provider list is done.

Reply:
I don't think we would need a clear button for this use case though.